### PR TITLE
Added drop-downs for heightmap sample channel selection to Parallax Mapping and Parallax Occlusion Mapping nodes

### PR DESCRIPTION
**Pull request by tracychen1234**
---
### Purpose of this PR
Fix for https://fogbugz.unity3d.com/f/cases/1347270/

- Added channel selection dropdowns to Parallax Mapping node and Parallax Occlusion Mapping node so users can decide which channel to sample the heightmap. The default channel for Parallax Mapping is green and for Parallax Occlusion is red.
- Have consulted Alex for the UI changes.

image

---
### Testing status
Succeed in Local tests (Play&Edit)

---

### CODE REVIEW

**Review by joshua-davis — APPROVED** [review]
Did some local testing and it looked good. Could use some tests though.

File: com.unity.shadergraph/Editor/Resources/Styles/Controls/EnumControlView.uss [hunk 1]
```diff
@@ -16,9 +16,10 @@ EnumControlView {
 }
 
 EnumControlView > Label {
-    max-width: 94px;
+    max-width: 140px;
```
Comment (cdxntchou) [inline com.unity.shadergraph/Editor/Resources/Styles/Controls/EnumControlView.uss:19]: Does this still look good on the other enum controls on other nodes?

**Review by cdxntchou — APPROVED** [review]

File: com.unity.shadergraph/Editor/Resources/Styles/Controls/EnumControlView.uss (hunk 1 above)
Comment (bencloward) [reply com.unity.shadergraph/Editor/Resources/Styles/Controls/EnumControlView.uss:19]: I looked at all of the nodes in the library both before and after this change. Although there are several nodes that have crowded or overlapping drop-down descriptions (UV, Scene Depth, Gradient Noise, Simple Noise) none of them appear to have changed based on this.  They're the same both with and without it.

Comment (tracychen1234) [reply com.unity.shadergraph/Editor/Resources/Styles/Controls/EnumControlView.uss:19]: Thank you, Ben! I also poped some nodes that have enum controls and they all look fine.
image

**Pull request merged**